specification/monitor/data-plane/Microsoft.Insights/preview/2023-03-01-preview/metricBatch.json

```diff
@@ -0,0 +1,484 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Monitor Metric Batch",
+    "version": "2023-03-01-preview"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{baseUrl}",
+    "useSchemePrefix": false,
+    "positionInOperation": "first",
+    "parameters": [
+      {
+        "name": "baseUrl",
+        "description": "The regional endpoint to use, for example https://eastus.metrics.monitor.azure.com. The region should match the region of the requested resources.",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      }
+    ]
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/metricBatch": {
+      "post": {
+        "tags": [
+          "Metrics"
+        ],
+        "operationId": "Metrics_Batch",
+        "description": "**Lists the metric values for multiple resources**.",
```

Comment:
This may not be processed as markdown - definitely won't for SDK code generation. Is there some reason the entire thing besides the trailing period is bolded?